package.json

```diff
@@ -79,7 +79,7 @@
   },
   "engines": {
     "node": ">=18",
-    "pnpm": ">=9.0.0-beta.2"
+    "pnpm": ">=9.0.1"
```

Comment:
did we have a version before?

Reply:
yes, it was added in this [commit](https://github.com/upleveled/preflight/commit/48967763de2c0cc66049377833dad5ae01b1b57f) with `v9.0.0-beta.3`, there was no `pnpm` version in the engine section. I removed it again 
[8b91f0c](https://github.com/upleveled/preflight/pull/525/commits/8b91f0cc9596eca7faae6a58efb97fd860640f52)